Sourcery/Parsing/FileParser.swift

```diff
@@ -250,9 +250,30 @@ final class FileParser {
 // MARK: - Details parsing
 extension FileParser {
 
+    fileprivate static let reservedKeywords = ["associatedtype", "class", "deinit", "enum", "extension",
```

Comment:
it's ok for now but we should just make the list available in the StencilSwiftKit